src/ResourceManager/Network/Commands.Network/Cortex/VirtualWan/P2sVpnServerConfiguration/UpdateAzureRmVirtualWanP2sVpnServerConfigCommand.cs

```diff
@@ -0,0 +1,227 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System.Security;
+using Microsoft.Azure.Commands.Network.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.WindowsAzure.Commands.Common;
+using MNM = Microsoft.Azure.Management.Network.Models;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet("Update",
+        ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "P2sVpnServerConfiguration",
+        SupportsShouldProcess = true),
+        OutputType(typeof(PSP2SVpnServerConfiguration))]
+    public class SetAzureRmVirtualWanP2sVpnServerConfigCommand : VirtualWanBaseCmdlet
+    {
+        [Alias("ResourceName", "P2sVpnServerConfigurationName")]
```

Comment:
This is also not the right parameter sets.  Please compare your help file to the cmdlet design review and update so they match.

Reply:
Updated. Please take a look.